pandas/tests/generic/test_series.py

```diff
@@ -52,33 +52,42 @@ def test_nonzero_single_element(self):
         s = Series([False])
         assert not s.bool()
 
+    @pytest.mark.parametrize(
```

Comment:
can you just put the Series *iniside* the test functions. (for all of these cases)

Reply:
I'm not sure I understand, what does it means "inside" the test functions?
I would really appreciate if you can post of example code for what you ment:)

Reply:
the parameters should be like `[np.nan, pd.NatT]` then simply construct what you need in the function